src/constants/theme.js

```diff
@@ -68,6 +68,10 @@ const fonts = {
   error: {
     fontFamily: 'red-hat-display-medium',
     fontSize: sizes.error
+  },
+  redError: {
+    fontFamily: 'open-sans',
+    fontSize: sizes.error
```

Comment:
There is no indication that this constant here defines a red color other than it's name, it's also hard to change, for example let's say that a designer later wants all errors to be dark red or brown, this change would cause the `redError` to be another color, it would be confusing to use and will cause changes in any place it is used. Basically prefer more abstract names, which are not described by it's properties but by it's purpose.

Also `open-sans` doesn't exist in Android, use the `error` which has `red-hat-display`.